Hardware/Hardware/CardReader/CardReaderEvent.swift

```diff
@@ -1,5 +1,26 @@
+/// An event dispatched by  a connected reader.
+/// It wraps an event type and an optional user-facing message
+public struct CardReaderEvent {
+    public let type: CardReaderEventType
+    public let message: String?
+
+    init(type: CardReaderEventType, message: String? = nil) {
```

Comment:
Should this also be public?

Reply:
Not really. This can only be initialized within the integration with the Stripe SDK, so I think it's better if it stays internal. Am I missing something?